### PR TITLE
Ful corrections of the review.

### PR DESCRIPTION
**Pull request by williamatr**
We do all the correction of the web https://validator.w3.org/, and add a comment in the readme abaut the logo of the web clone.

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
# Hi team,👋👋
👌 job but we still need to improve a few things before merging

 - You should fix the errors given to you by the[ validator ](https://validator.w3.org)when uploading the index file.🙂

- It has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar 🙂

- It has a subscribe button
> currently what you have there is an image.

- It’s better to delete commented code check line 23,51,52,102 of ```index.html```

**[optional]**
We highly recommend the usage of semantic HTML for each project. In fact, it is actually a requirement in some of your subsequent projects. I suggest you use it in the next.


### Kindly make those changes and submit for review

You are almost there Guys,
Happy coding

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+However we decided to avoid using the exact same graphic elements as youtube, and do
+something really similar, hence the U-Tube "branding" all the icons were designed by us
+and so was the U-Tube logo. We took this freedom in order to make the task a little more
+fun, and we guessed that this slight deviation wouldn't be a problem.
+
+This project is based on this: https://www.theodinproject.com/courses/html5-and-css3#basic-html-page-structure
+
+After our second review, we were told to make it look exactly as the youtube page, using their logos and so.
+
+Sadly, we were right in our assumption that is was the best to create our own logo and images, since we contacted youtube (mexico) and were denied authorization to use their branding an immage, please refrain from asking us to violate copyright law. This exercise goals are about doing an structurally similar webpage, not about using a thrid party's intelectual property.
```
Comment (bolah2009) [inline README.md]: #### `HINT`

Add a descriptive readme. Details like assignment link (from Odin), live link (e.g. from [githack](https://raw.githack.com/)), screenshot and what was learned during the project can make the project more descriptive.

File: index.html [hunk 2]
```diff
@@ -1,213 +1,340 @@
...
+            </div>
+ </nav>
+  <main>
+    <div class="wrapper">
+      <div id="videonstuff">
+         <!-- Video Blue -->
+          <div class="video">
+              <video poster ="./img/video_thumbnail.png" src="https://www.youtube.com/embed/oXMb1mhIpqQ" controls>
 
+              </video>
```
Comment (bolah2009) [inline index.html:42]: Please let's embed an actual video. Remember the main objective of this project is embedding images and video.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Awesome job Team :smile: 

## [The clone](https://raw.githack.com/williamtovar/Yutube_FerandWill/cd8b98a791de62eaba7ab5f823a8480e1bae7fd2/index.html) looks good to me using HTML image effectively.

However, let's check the following:
- [REQUIRED] There should be an embedded video in the clone
- [REQUIRED] Use descriptive readme (Let's move comments about reviews to this PR (Pull Request) comments)
- Resolve merge conflicts
- Please try and use one PR throughout your review process, it makes it for TSEs  to follow-up with previous reviews easily

#### :recycle: Changes required

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Review by uche-inyama — COMMENTED** [review]
## Hi team, good job

Kindly update your README as my colleague suggested, go through this [link](https://github.com/othneildrew/Best-README-Template), use it as a reference/guide; implement what fits your project requirement or what you have done in this project. :slightly_smiling_face: 

## After implementing the change, submit another form for a code review.

#### ``` Have a great day``` :slightly_smiling_face:

**Review by stratospherique — COMMENTED** [review]
### @williamtovar :wave: 

- Since you have merged this pull request and still have yet to apply reviewers suggestion about the README.md file. I suggest that you open a new pull request, make the changes and ask for code review again :pray: .

##### Happy coding :smiley_cat:

**Pull request merged**